tasks/internal/create-advisory-task/create-advisory-task.yaml

```diff
@@ -273,11 +273,13 @@ spec:
           fi
           ADVISORY_NUM=$(printf "%04d" "$LIVE_ID")
 
-          # Check if the advisory number is already used
-          FIND_RESULT=$(find data/advisories -type d -path "*/${YEAR}/${ADVISORY_NUM}")
-          if [[ -n "${FIND_RESULT}" ]]; then
+          # Check if the advisory number is already used;
```

Comment:
```suggestion
          # Check if the advisory number is already used
```

Reply:
Good catch. I was adding more comments in a previous version, but ended up not needing them. Fixed.